### PR TITLE
Add noscript tag for disabled JavaScript

### PR DESCRIPTION
**Pull request by Leviakc**
Sin `<noscript>`
Selección_009

Con `<noscript`
Selección_010

### CODE REVIEW

**Comment by Leviakc** [comment]
> Good thinking, but I think it would be necessary to modify the design to something more appropriate to the style of the website to avoid looking like a shitpost.

Yes, I was thinking the same, but couldn't find a right background and text color also I was trying to make a little funny haha but I'll change it!

**Comment by Zyruks** [comment]
BunniesWhatGIF

**Comment by afordigital** [comment]
OWW! I love the idea of adding a noscript tag for disabled JavaScript, I also agree with the idea of make it looks more like the page design, if we approach the design a little, I'll merge!

File: index.html [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
+  <div id="root"></div>
+  <script type="module" src="/src/main.tsx"></script>
+  <noscript>
+    <style>
+      body {
+        background-color: #0e7490aa;
+        min-height: 100svh;
+        display: grid;
+        place-content: center;
+      }
```
Comment (Zyruks) [inline index.html:21]:
```suggestion
      body {
        background-color: #0A0A0A;
        min-height: 100svh;
        display: grid;
        place-content: center;
        color: #F5F5F5;
      }
```

Mejoras de estilo

File: index.html [hunk 2]
```diff
@@ -1,13 +1,49 @@
...
+
+      .main {
+        margin-inline: 1rem;
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+      }
```
Comment (Zyruks) [inline index.html:28]:
```suggestion
     .main {
        text-align: center;
        background-color: #0d0d0d;
        border-radius: 0.7rem;
        padding: 2.5rem;
      }
```

File: index.html [hunk 3]
```diff
@@ -1,13 +1,49 @@
...
+
+      .paragraph {
+        font-size: 2rem;
+        color: #292524;
+        text-align: center;
+      }
```
Comment (Zyruks) [inline index.html:34]:
```suggestion
      .paragraph {
        font-size: 1.4rem;
        margin-block-end: 1.5rem;
      }
```

File: index.html [hunk 4]
```diff
@@ -1,13 +1,49 @@
...
+
+      img {
+        max-width: 100%;
+      }
+    </style>
+    <main class="main">
+      <p class="paragraph">Lo siento, necesitas activar JavaScript para ver la
+        página 🥺 </p>
+      <img src="/images/hamtafor.webp">
```
Comment (Zyruks) [inline index.html:43]:
```suggestion
      <img src="/images/hamtafor.webp" alt="Afordin Hamster version">
```

Agregar el alt :3

**Review by Zyruks — CHANGES_REQUESTED** [review]
With these changes it would look like this:

chrome_April_01_2024_wwbL

File: index.html (hunk 4 above)
Comment (afordigital) [reply index.html:43]: Good feedback! I'd also delete the 'atte afor digital' and I would only put the name in the alt image

**Comment by Leviakc** [comment]
**This would be the final design, I think with no image looks a little more profesional, what are your thoughts?**
Selección_015

**Comment by afordigital** [comment]
I really like it! its a very good contribution, thank u a lot (also love the hamstercito)! GOOD JOB! 😁

**Pull request merged**